sklearnex/dispatcher.py

```diff
@@ -143,6 +146,10 @@ def get_patch_map_core(preview=False):
         )
         from .linear_model import LinearRegression as LinearRegression_sklearnex
         from .linear_model import LogisticRegression as LogisticRegression_sklearnex
```

Comment:
Absence of decorated Lasso, Ridge, ElasticNet in sklearnex patch_map creates uncertainty: is decorated sklearnex class used or undecorated daal4py one?

Reply:
See https://github.com/intel/scikit-learn-intelex/pull/1832#discussion_r1648578243 . Thank you!